linalg/operator.hpp

```diff
@@ -338,7 +338,7 @@ class Operator
       3. F(u,k,t) = M k - g(u,t) and G(u,t) = 0.
 
     Note that depending on the ODE solver, some of the above choices may be
-    preferable to the others.
+    preferable to the others. See ImplicitSolve() documentation for more details.
```

Comment:
To me, this comment gives the impression that the choice of $F$ and $G$ only impacts `ImplicitSolve` ... Why not leave the original comment as-is (indicating the choice of $F$ and $G$ impacts many functions throughout `TimeDependentOperator`?  I'm probably missing something simple.